cedar-policy-validator/src/validation_result.rs

```diff
@@ -149,18 +149,18 @@ pub enum ValidationErrorKind {
     /// applied to a principal and resources that both satisfy their respective
     /// head conditions.
     #[error(
-        "Unable to find an applicable action given the policy head constraints{}{}",
+        "unable to find an applicable action given the policy head constraints{}{}",
         if .0.would_in_fix_principal { ". Note: Try replacing `==` with `in` in the principal clause" } else { "" },
         if .0.would_in_fix_resource { ". Note: Try replacing `==` with `in` in the resource clause" } else { "" }
     )]
     InvalidActionApplication(InvalidActionApplication),
-    /// A type error was found by the type checker.
+    /// The type checker found a type error.
```

Comment:
[nit] could probably just say "an error". We already said the type checker found it.

Reply:
Good point. Changed.